src/python/libcgroup.pyx

```diff
@@ -310,6 +318,43 @@ cdef class Cgroup:
             i = i + 1
         return mount_points
 
+    @staticmethod
+    def create_scope(scope_name, pid=None, slice_name='libcgroup.slice', delegated=True,
+                     systemd_mode=SystemdMode.CGROUP_SYSTEMD_MODE_REPLACE):
+        """Create a systemd scope
+
+        Arguments:
+        scope_name - name of the scope to be created
+        pid - pid of the process to place in the scope.  If None is provided, libcgroup will
+              place a dummy process in the scope
+        slice_name - name of the slice where the scope should reside
+        delegated - if true, then systemd will not manage the cgroup aspects of the scope.  It
+                    is up to the user to manage the cgroup settings
+        systemd_mode - setting to tell systemd how to handle creation of this scope
```

Comment:
Does including conflict resolution to the `systemd_mode` make sense, something  like `systemd_mode - setting to tell systemd how to handle the creation of this scope and resolve conflict if the scope and slice exist`

Reply:
Good improvement.  Done